packages/react/__tests__/src/components/Select/index.js

```diff
@@ -29,10 +29,9 @@ const withCustomOptions = (otherProps = {}) => {
 test('renders the expected UI', () => {
   const wrapper = withCustomOptions();
 
-  expect(wrapper.find('.Field__select--label')).toBeTruthy();
-  expect(wrapper.find('.Field__select--required')).toBeTruthy();
-  expect(wrapper.find('.Field__select')).toBeTruthy();
-  expect(wrapper.find('.Field__option')).toBeTruthy();
```

Comment:
These assertions weren't actually doing anything before, since the enzyme wrapper is technically "truthy".

The --required in particular actually should _not_ exist, and we are asserting as much on line 35